### PR TITLE
Update links to Data Tracker on Project Summary page

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/8251

## Testing
**URL to test:** 
https://8251-data-tracker-links--atd-moped-main.netlify.app/

**Steps to test:**

Note: You need to be signed into Data Tracker to be able to follow the links to work orders and signals

https://8251-data-tracker-links--atd-moped-main.netlify.app/moped/projects/383/

This is a project that has already been linked to Data Tracker. You can see the link to the signal as well as a link to the Work Orders. 
Screen Shot 2022-02-02 at 1 02 20 PM

If a project hasn't been synced yet, there is no link to the work orders, and there is a link to synchronize the project.
Screen Shot 2022-02-02 at 12 59 12 PM

If a project is synced, but there are no signals on the project, the link will still say "synchronize with data tracker" since there are no signals to link to. 

---
#### Ship list
- [ ] Code reviewed 
- [x] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)

### CODE REVIEW

**Review by amenity — APPROVED** [review]
This all looks great to me, @chiaberry ... Assigning @johnclary for review as well.

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryKnackDataTrackerSync.js [hunk 1]
```diff
@@ -181,32 +190,23 @@ const ProjectSummaryKnackDataTrackerSync = ({
...
+                // otherwise render link to synchronize with knack
                 <>
                   <Link
                     className={classes.fieldLabelText}
                     onClick={() => {
                       handleSync();
                     }}
                   >
-                    {"Synchronize"}
+                    {"Synchronize with Data Tracker"}
```
Comment (frankhereford) [inline moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryKnackDataTrackerSync.js:209]: Thank you for cleaning this up! 🙏

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js [hunk 2]
```diff
@@ -93,7 +93,7 @@ const ProjectSummaryProjectDescription = ({
               fullWidth
               multiline={true}
               rows={4}
-              id="moped-project-website"
+              id="moped-project-description"
```
Comment (frankhereford) [inline moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectDescription.js:96]: 👍  Thank you

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectECapris.js [hunk 3]
```diff
@@ -159,7 +159,7 @@ const ProjectSummaryProjectECapris = ({
           <>
             <TextField
               fullWidth
-              id="moped-project-website"
+              id="moped-project-ecapris"
```
Comment (frankhereford) [inline moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryProjectECapris.js:162]: 👍  again!

**Review by frankhereford — APPROVED** [review]
This is great work. I've gone over the code as well as tested it on the PR deployment. @chiaberry, I'm really grateful for the work you've done on the knack synchronization UI elements as well as the code organization and readability. I'm looking forward to working from this!

🚢 🚢 🚢

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryWorkOrders.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+              <OpenInNew className={classes.linkIcon} />
+            </Link>
+          }
+          classes={classes}
+          spanClassName={classes.fieldLabelTextSpanNoBorder}
+        />
+      </Box>
+    </Grid>
+  ) : (
+    // If there is no knack project url, render an empty grid space
```
Comment (mateoclarke) [inline moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryWorkOrders.js:53]: 👍

**Pull request merged**